src/components/global/localNav/localNav.stories.js

```diff
@@ -38,50 +46,234 @@ export const LocalNav = {
       menu: 'Menu',
     },
   },
-  render: (args) => html`
-    <kyn-local-nav
-      ?pinned=${args.pinned}
-      pinText=${args.pinText}
-      unpinText=${args.unpinText}
-      .textStrings=${args.textStrings}
-    >
-      <kyn-local-nav-link href="javascript:void(0)" active>
-        <kd-icon slot="icon" .icon=${sampleIcon}></kd-icon>
-        Link 1
-      </kyn-local-nav-link>
-
-      <kyn-local-nav-link href="javascript:void(0)">
-        <kd-icon slot="icon" .icon=${sampleIcon}></kd-icon>
-        Link 2
-
-        <kyn-local-nav-link slot="links" href="javascript:void(0)">
+  render: (args) => {
+    return html`
+      <kyn-local-nav
+        ?pinned=${args.pinned}
+        pinText=${args.pinText}
+        unpinText=${args.unpinText}
+        .textStrings=${args.textStrings}
+      >
+        <kyn-local-nav-link href="javascript:void(0)" active>
           <kd-icon slot="icon" .icon=${sampleIcon}></kd-icon>
-          L2 Link 1
+          Link 1
         </kyn-local-nav-link>
-        <kyn-local-nav-link slot="links" href="javascript:void(0)">
-          <kd-icon slot="icon" .icon=${sampleIcon}></kd-icon>
-          L2 Link 2
-        </kyn-local-nav-link>
-      </kyn-local-nav-link>
 
-      <kyn-local-nav-link href="javascript:void(0)" expanded>
-        <kd-icon slot="icon" .icon=${sampleIcon}></kd-icon>
-        Link 3
-
-        <kyn-local-nav-link slot="links" href="javascript:void(0)">
+        <kyn-local-nav-link href="javascript:void(0)">
```

Comment:
i assume that these <kyn-local-nav-links> are repeated for readability and because this is the storybook file? that would make sense. otherwise, using something like the `renderLink` function to DRY this up would be good.

Reply:
Mainly so the example code is easy to see in storybook. If we start abstracting it, it makes it hard to find. You're right though, most likely the application will render links in a loop rather than hard-code each tag.